vortex-flatbuffers/flatbuffers/vortex-array/array.fbs

```diff
@@ -37,9 +35,10 @@ table ArrayNode {
 }
 
 table ArrayStats {
-    min: ScalarValue;
-    max: ScalarValue;
-    sum: ScalarValue;
+    /// Protobuf serialized ScalarValue
+    min: [ubyte];
```

Comment:
Single element struct with `protobytes: [ubyte]` I think?

Reply:
this is sadly not valid flatbuffer sadly